radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/core/handlers/EngineConfigurationHandler.java

```diff
@@ -78,18 +78,18 @@
 import com.radixdlt.statecomputer.checkpoint.Genesis;
 import com.radixdlt.statecomputer.forks.ForkConfig;
 import com.radixdlt.utils.Bytes;
-import java.util.TreeMap;
+import java.util.NavigableMap;
 
 public final class EngineConfigurationHandler
     extends CoreJsonRpcHandler<EngineConfigurationRequest, EngineConfigurationResponse> {
-  private final TreeMap<Long, ForkConfig> forks;
+  private final NavigableMap<Long, ForkConfig> forks;
```

Comment:
why is it necessary?

Reply:
this map is going away with the coordinated forks branch

Reply:
See above, this change was made as part of cleanup of code smell.